src/helpers/makeUpperCase.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Generic to lower case
+ * @function
+ *
+ * @param {string} word
+ *
+ * @returns {Function} call to turn upper case to lower case
+ * @description turn upper case to lower case
+ */
+const makeUpperCase = (word) => {
+  const formattedWord = word
+    .toLowerCase()
+    .split(' ')
+    .map(f => f.charAt(0).toUpperCase() + f.substring(1))
+    .join(' ');
+  return formattedWord;
```

Comment:
The team convention calls for a new line before `return` statements. Kindly adhere to it